commit 9a19c9b3cff593be15ebf12f1350999e85c0fec8
Author: ptrcnull <git@ptrcnull.me>
Date:   Thu Feb 3 05:19:42 2022 +0100

    add quotes to "unexpected token" message
    
    that seems to come from Crystal itself,
    Alpine has Crystal 1.3.2 whereas Ameba's CI uses 1.2.2

diff --git a/spec/ameba/rule/lint/syntax_spec.cr b/spec/ameba/rule/lint/syntax_spec.cr
index f7402e7..c362af9 100644
--- a/spec/ameba/rule/lint/syntax_spec.cr
+++ b/spec/ameba/rule/lint/syntax_spec.cr
@@ -31,7 +31,7 @@ module Ameba::Rule::Lint
 
       issue.rule.should_not be_nil
       issue.location.to_s.should eq "source.cr:1:11"
-      issue.message.should eq "unexpected token: end (expected ';' or newline)"
+      issue.message.should eq "unexpected token: \"end\" (expected \";\" or newline)"
     end
 
     it "has highest severity" do
